browser/scripts/worldEditor/worldEditor.js

```diff
@@ -257,7 +263,7 @@ WorldEditor.prototype.setSelection = function(selected) {
 	if (!anySelected)
 		this.clearSelection()
 
-	E2.ui.emit('worldEditor:selectionSet')
+	this.emit('selectionSet', this.getSelectedNodes())
```

Comment:
This is local. the UI store does not mediate in the work loop. Of course the worldEditor could dispatch so that the other side knows what you've selected something, but that's outside my UI scope for this task. I think worldEditor emitting on itself is now less risky to cascade than previously emitting on E2.ui, which worldEditor shouldn't know anything about.